typings/omni-fetch.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'omni-fetch' {
+  export default fetch;
```

Comment:
Since pretend isn't using isomorphic fetch directly anymore with this PR please remove isomorphic fetch and add typings for omni-fetch.